src/components/common/MwInput/MwInput.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ChangeEventHandler } from 'react';
+
+import './MwInput.scss';
+
+export interface MwInputProps {
+  className: string;
+  containerStyle: string;
+  name: string;
+  onChange: ChangeEventHandler;
+  type: string;
+  value: string;
+  placeholder?: string;
+  error?: string;
+}
+
+const MwInput = (props: MwInputProps): JSX.Element => {
+  const inputRef = React.useRef(null);
+
+  const handleClick = (): void => {
+    if (inputRef && inputRef.current) {
+      inputRef.current.focus();
+    }
+  };
+  return (
+    <>
```

Comment:
The fragment is not needed, since there is parent `div` on line 26

Reply:
thanks